LaTeX/LaTeX.sublime-syntax

```diff
@@ -102,6 +102,15 @@ contexts:
         - newcommand-definition
         - newcommand-commandname
 
+    - match: '(\\)(?:(?:New|Renew|Provide|Declare)(:?Expandable)?DocumentCommand)'
```

Comment:
Maybe

1. keep unquoted for consistency reasons
2. append word break pattern

```suggestion
    - match: (\\)(?:(?:New|Renew|Provide|Declare)(:?Expandable)?DocumentCommand){{endcs}}
```

or

```suggestion
    - match: (\\)(?:(?:New|Renew|Provide|Declare)(:?Expandable)?DocumentCommand)(?:\*|(?![A-Za-z@]))
```